src/compiler.jl

```diff
@@ -4366,6 +4366,9 @@ function GPUCompiler.codegen(output::Symbol, job::CompilerJob{<:EnzymeTarget};
 
     mod, meta = GPUCompiler.codegen(:llvm, primal_job; optimize=false, toplevel=toplevel, cleanup=false, validate=false, parent_job=parent_job)
     prepare_llvm(mod, primal_job, meta)
+    for f in functions(mod)
+        permit_inlining!(f)
+    end
 
     LLVM.ModulePassManager() do pm
         API.AddPreserveNVVMPass!(pm, #=Begin=#true)
```

Comment:
@vchuravy @simsurace so I actually disagree with this part of the autoformatter, since I would very much like to know what that arg corresponds to (as presently specified with a comment).

What's the right way to do so in-format?

Reply:
Hmm, good question. It seems to be a known issue https://github.com/domluna/JuliaFormatter.jl/issues/562